### ISSUE TITLE
Uncaught TypeError: Cannot read property 'pid' of null

### ISSUE THREAD

**Issue by provrico** [lsegal/atom-runner#107]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.3.2
**System**: Mac OS X 10.11.2
**Thrown From**: [atom-runner](https://github.com/lsegal/atom-runner) package, v2.5.0

### Stack Trace

Uncaught TypeError: Cannot read property 'pid' of null

```
At /Users/ricoordonio/.atom/packages/atom-runner/lib/atom-runner.coffee:162

TypeError: Cannot read property 'pid' of null
    at ChildProcess.<anonymous> (/Users/ricoordonio/.atom/packages/atom-runner/lib/atom-runner.coffee:162:18)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at maybeClose (internal/child_process.js:817:16)
    at Socket.<anonymous> (internal/child_process.js:319:11)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at Pipe._onclose (net.js:469:12)
```

### Commands

```
     -0:00.6.0 run:file (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
atom-runner, v2.5.0

# Dev
No dev packages
```

**Comment by lsegal** [comment]
Can you provide actual steps to reproduce? Specifically, does this happen when an error in the script you're running occurs?

**Comment by provrico** [comment]
This error occurred right after upgrading atom runner. It could not find the path for node.
I was able to fix it by reinstalling node.

**Comment by lsegal** [comment]
Thanks I know what's causing this. Fix coming.

**Comment by lsegal** [comment]
Fixed, tagged, and published as 2.5.1. Thanks for reporting.
